### PR TITLE
Remove former maintainers

### PR DESCRIPTION
**Pull request by therealmitchconnors**
The most common reason to send a PR to this repository is to join the Istio org.  If you are joining, please fill out this template.  If you are not, please **delete all the below**.

- [ ] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [ ] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [ ] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [ ] I have joined the [Istio discussion board](https://discuss.istio.io) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
- [ ] I have joined [Istio's Slack workspace](https://slack.istio.io)

List your company name, or indicate Individual if you're not affiliated with a company:

Provide a link to at least one PR that you have successfully pushed to one of the Istio repos:

### CODE REVIEW

**Review by ericvn — COMMENTED** [review]
Thanks for doing this Mitch.

**Comment by kfaseela** [comment]
LGTM

File: org/teams.yaml [hunk 1]
```diff
@@ -407,7 +374,6 @@ teams:
         description: Release managers for Istio 1.9
         members:
         - jacob-delgado
-        - shamsher31
```
Comment (kfaseela) [inline org/teams.yaml:410]: why do we have to remove from here?

Comment (linsun) [reply org/teams.yaml:410]: +1

**Comment by linsun** [comment]
LGTM, @kfaseela's comment is valid on prior release mgrs. 

For the record, prior release mgrs are doc-ed here: https://github.com/istio/community/blob/master/RELEASE_MANAGERS.md

**Review by kfaseela — APPROVED** [review]

**Pull request merged**